Userland/Utilities/run-tests.cpp

```diff
@@ -322,9 +322,17 @@ int main(int argc, char** argv)
     if (config->num_groups() == 0)
         warnln("Empty configuration file ({}) loaded!", config_file.is_empty() ? "User config for Tests" : config_file.characters());
 
-    if (exclude_pattern.is_empty())
-        exclude_pattern = config->read_entry("Global", "NotTestsPattern", "$^"); // default is match nothing (aka match end then beginning)
-
+    if (exclude_pattern.is_empty()) {
+        StringBuilder builder;
+        builder.append(config->read_entry("Global", "NotTestsPattern", "$^")); // default is match nothing (aka match end then beginning)
+        for (auto regex : config->read_entry("Global", "SkipRegex", "$^").split_view(' ')) {
```

Comment:
This feels kinda weird, now there's three ways to mark a test as "skip"

SkipTests takes a space separated list of tests to skip

SkipDirectories takes a space separated list of directories to skip

And

SkipRegex takes a path regex to.. ignore? But your comment on the PR says these are different from ignore? You're adding them to the ignore regex 🤔. Not the skip regex. What's the goal here again? 😅

Reply:
The difference was supposed to be the name/intention, but good catch, I will change that

Reply:
The phrasing of "ignore" vs "skip" seems obvious in hindsight, idk why I didn't think of it the first go around. If you wanted to change NotTestPattern to IgnoreRegex that would probably be a lot clearer.

Reply:
Well it should still skip the ue specific test,
and while trying to implement it I somehow broke something with the regex byte-code engine, so will look into it tomorrow

Reply:
This should be fixed now